### ISSUE TITLE
Please add instructions to be able to log in to a Deskop Environment

### ISSUE THREAD

**Issue by tgonzalez89** [GoDzM4TT3O/Arch#1]
First of all, thanks for this guide. It's awesome!
I'd like to see the instructions to be able to launch the different desktop environments and be able to get to the graphical log in screen by default. I'm playing around with the options, but I haven't been able to make it work. (first time using Arch and installing a DE from scratch)

**Comment by saloniamatteo** [comment]
Thanks! :) 

I will add instructions later, as you requested. Stay tuned!

Also, what DE are you looking to install specifically?

**Comment by saloniamatteo** [comment]
Reopening to tell you this: if you have any other doubts, please comment here, otherwise close this issue.

**Comment by tgonzalez89** [comment]
I was trying out LXQt, but I just tried that one out arbitrarily. I'm actually more interested in MATE or GNOME.
Anyway, I see you added instructions to be able to start the DE, nice. Thanks!
Could you add instructions to be able to boot directly to the DE? Or to boot to a login screen where you can choose your DE. I remember installing Debian and after you installed a DE I was able to directly boot to a login screen and choose between the different DEs I had installed.

**Comment by tgonzalez89** [comment]
In my case I managed to install lightdm:
`sudo pacman -S lightdm lightdm-gtk-greeter xorg-server`
`sudo systemctl enable lightdm.service`

Another suggestion: audio support.
`sudo pacman -S pulseaudio pulseaudio-alsa`
Extra steps might be needed for a specific DE, in my case I also installed `xfce4-pulseaudio-plugin`. (I switched now to xfce). But I think this is DE specific and I don't know if you want to get into that.
